src/console/Normalize/Processor.cs

```diff
@@ -54,27 +54,45 @@ public async Task ConsumeAsync(IEnumerable<IEventMessage> events)
                 IomtMetrics.DeviceEvent(),
                     events.Count());
 
+            var eventBatchSizeBytes = 0;
```

Comment:
>eventBatchSizeBytes [](start = 16, length = 19)

Can we use the Event Processing Meter here as well?  I like how you did that an it is a shame we aren't using it here.  This should technically perform better but I like the cleaner approach and the extra overhead shouldn't be too much now that we switched from JSON serialization.

Reply:
Yep. I didn't use the meter due to the slight performance improvement from not having to iterate again, but since it is only a slight improvement I went ahead and changed console app to now use the Event Processing meter.